chain/x/zoracle/internal/keeper/request.go

```diff
@@ -121,6 +121,21 @@ func (k Keeper) CheckRequestExists(ctx sdk.Context, id int64) bool {
 	return store.Has(types.RequestStoreKey(id))
 }
 
+// ConsumeGasForExecute saves and consumes gas from the current context to use in executing
+// at end block time.
+func (k Keeper) ConsumeGasForExecute(ctx sdk.Context, id int64, executeGas uint64) sdk.Error {
+	request, err := k.GetRequest(ctx, id)
+	if err != nil {
+		return err
+	}
+
+	ctx.GasMeter().ConsumeGas(executeGas, "Execute gas")
```

Comment:
I think the convention is to do dash? "execute-gas"

Reply:
Change to "ExecuteGas"